earthpy/mask.py

```diff
@@ -21,11 +22,13 @@ def make_cloud_mask(mask_arr, vals):
         True (Boolean)
     """
 
-    # Improved by Joe! Not tested yet but should work!
+    # construct the mask
```

Comment:
So much cleaner now! @joemcglinchy will you capitalize each comment line for PEP 8 standards so we're consistent?